### ISSUE TITLE
Feature Request: Column Filters / Search

### ISSUE THREAD

**Issue by sagansapien** [NUWCDIVNPT/stig-manager#51]
Lots of various columns would be more useful if there were a filter option, similar to Excel.

Potential locations for filters:

Collections > Collection A > STIGs > STIG A > Collection Review > "Reviews of <rule>" > Status, Result columns

Collections > Collection A > Reports > Findings > Aggregated Findings > CAT, STIGs columns

This would help in reviewing large quantities of CKLs and looking for specific attributes, such as NR, Open, etc. findings.

Thanks!

**Comment by cd-rite** [comment]
Agreed, that would be useful for review. Right now, columns can be sorted on those attributes, but display would still be pretty cluttered with large number of reviews. I've added this to our Enhancement backlog. We are currently in a bit of a bug-fixing cycle, but will address User Stories like this soon. Thanks for contributing.

**Comment by sagansapien** [comment]
Thanks again!
